### PR TITLE
LPD-38392 Reindex ctEntries

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-38392

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:publications

**Comment by brookedalton** [comment]
ci:test:publications

File: modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/model/listener/CTCollectionModelListener.java [hunk 1]
```diff
@@ -58,6 +65,29 @@ public void onAfterUpdate(
...
 
+		if (!Objects.equals(
+				originalCTCollection.getName(), ctCollection.getName()) ||
+			(originalCTCollection.getStatus() != ctCollection.getStatus())) {
+
+			Indexer<CTEntry> indexer = _indexerRegistry.getIndexer(
+				CTEntry.class);
+
+			if (indexer != null) {
+				for (CTEntry ctEntry :
```
Comment (david-truong) [inline modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/model/listener/CTCollectionModelListener.java:76]: @brookedalton you can pass the whole list into reindex without needing to loop

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
@brookedalton can you explore why the tests failed?

**Comment by david-truong** [comment]
https://github.com/liferay-site-management/liferay-portal/pull/1475

**Pull request closed without merging**